### ISSUE TITLE
Screen reader skips content

### ISSUE THREAD

**Issue by danielghost** [adaptlearning/adapt-contrib-narrative#243]
### Subject of the issue/enhancement/features
JAWS (doesn't seem to occur with NVDA) can intermittently skip chunks of content when focus is applied to the display text (via `a11y.focusFirst`) after using the navigation controls.

### Your environment
* Framework v5.22.11
* Chrome

### Steps to reproduce
Using the `DownArrow`, select `narrative__controls-right`. If the display text contains enough text (or multiple paragraphs) for a screen reader to focus different lines when navigating via the down arrow, it may skip the start of the content and begin reading from further down. If `narrative__controls-right` is selected via `Tab`, the screen reader always seems to start reading from the beginning.

**Comment by danielghost** [comment]
Seems like this is a known JAWS issue and cannot be resolved in Adapt:
https://github.com/FreedomScientific/VFO-standards-support/issues/491

Adding the option to defer the [focus](https://github.com/adaptlearning/adapt-contrib-narrative/blob/master/js/NarrativeView.js#L70) seems to mitigate the issue, but doesn't negate it completely:
```JavaScript
a11y.focusFirst($elementToFocus, { defer: true });
```

**Comment by oliverfoster** [comment]
Closing as external bug
